text/0006-architecture-review-process.md

```diff
@@ -41,14 +43,20 @@ The Review Packet includes:
     - what review and discussion of the proposed architecture has already happened (and with whom)
   * A competitive analysis suggesting alternatives, costs, and opportunities
 
-3. The chair creates a list of questions to be discussed during the review. These questions should be:
+3. The chair or reviewer creates a list of questions to be discussed during the review. These questions should be:
   * Open-ended
   * Made available to the the team prior to the review so they can prepare answers
 
-4. The chair convenes a review meeting. The discussions of this meeting should be carefully minuted. The meeting comprises:
-  * The team making the proposal
-  * The chair
-  * Other senior engineers who will provide valuable input
+4. The chair or reviewer convenes a review meeting. The discussions of this meeting should be carefully minuted.
+  * The meeting should have a limited attendance to avoid a sprawling unfocussed meeting, but it should include:
+    * The team making the proposal, the reviewer (and chair if used)
```

Comment:
Nit: rather than using "chair or reviewer" and caveating references to the chair with "if used", it'd be better to simply define the chair as a permanent role that might happen to be filled by the reviewer. Then you can simply use "chair" when you mean the chair specifically, and that'll apply even in cases where the role of chair is being filled by the reviewer.